docs/examples/sampling.rst

```diff
@@ -29,5 +29,5 @@ persistent_sampling
 persistent_sampling_var_resources
 ---------------------------------
 .. automodule:: persistent_sampling_var_resources
-  :members:
+  :members: uniform_sample_with_var_gpus
```

Comment:
We should include all the gen_f functions unless they are very obviously just for testing and/or would be misleading to users. It is good to have a long list of functions, not bad. There are some good examples of using variable resources here in this file.